packages/paste-website/src/pages/utilities/screen-reader-only/index.mdx

```diff
@@ -0,0 +1,212 @@
+  </Paragraph>
+  <Flex vAlignContent="center">
+    <Flex marginRight="space40">
+      <Button variant="primary">
+        Launch Flex
+      </Button>
+    </Flex>
+    <Anchor href="#">
+      View all projects
+    </Anchor>
+  </Flex>
+</Card>`}
+</LivePreview>
+
+### Repetitive buttons and hidden headers
+
+Each column in a table is required to have a column header. Sometimes those column headers can be visually hidden from sighted users if desired. To do so use the Screen Reader Only component to hide the cell content for that column header.
+
+Another common use case is for providing context to repetitive actions. Visually the "Buy" button is associated with the phone number via the row highlighting, but to a screen reader user the "Buy" action might be unclear. To provide additional clarification we can add _what_ you would be buying as visually hidden text to the button.
+
+<LivePreview scope={{Table, Thead, Tbody, Tr, Td, Th, Button, Text, ScreenReaderOnly}} language="jsx">
+  {`<Table>
+  <Thead>
+    <Tr>
+      <Th>Number</Th>
+      <Th>Type</Th>
+      <Th>Monthly Fee</Th>
+      <Th>
+        <ScreenReaderOnly>Actions</ScreenReaderOnly>
+      </Th>
+    </Tr>
+  </Thead>
+  <Tbody>
+    <Tr>
+      <Th>
+        <strong>+1 (903) 603-9469</strong>
+        <Text as="div" textColor="colorTextWeak" fontWeight="normal">
+          Mabank, TX
+        </Text>
+      </Th>
+      <Td>local</Td>
+      <Td>$1.00</Td>
+      <Td>
+        <Button variant="secondary" size="small">
+          Buy <ScreenReaderOnly>+1 (903) 603-9469</ScreenReaderOnly>
+        </Button>
+      </Td>
+    </Tr>
+    <Tr>
+      <Th>
+        <strong>+1 (470) 309-0026</strong>
+        <Text as="div" textColor="colorTextWeak" fontWeight="normal">
+          Adairsville, GA
+        </Text>
+      </Th>
+      <Td>local</Td>
+      <Td>$1.00</Td>
+      <Td>
+        <Button variant="secondary" size="small">
+          Buy <ScreenReaderOnly>+1 (470) 309-0026</ScreenReaderOnly>
+        </Button>
+      </Td>
+    </Tr>
+    <Tr>
+      <Th>
+        <strong>+1 (484) 662-4088</strong>
+        <Text as="div" textColor="colorTextWeak" fontWeight="normal">
+          Hamburg, PA
+        </Text>
+      </Th>
+      <Td>local</Td>
+      <Td>$1.00</Td>
+      <Td>
+        <Button variant="secondary" size="small">
+          Buy <ScreenReaderOnly>+1 (484) 662-4088</ScreenReaderOnly>
+        </Button>
+      </Td>
+    </Tr>
+  </Tbody>
+</Table>`}
+</LivePreview>
+
+## When to use Screen Reader Only
+
+Use the Screen Reader Only component when you are required to provide additional or contextual information to a screen reader user which a sighted user maybe able to infer and when you cannot do so via semantic HTML.
+
+<DoDont>
+  <Do
+    title="Do"
+    body="Use to provide additional text content to users of assistive technology"
+    preview={false}
+  />
+  <Dont title="Don't" body="Don't use to hide interactive content" preview={false} />
+</DoDont>
```

Comment:
Any worth to adding a Do/Dont about:
- Adding meaningful text that relates to the what Screen Reader Only is added to
- Dont add spammy text

Reply:
I like this, but we should add an example.

Reply:
This might be difficult to do an example of. Are you suggesting it shouldn't be used for stuff like old school SEO keyword stuffing, @richbachman

Reply:
Yeah. Thats what I was thinking. Might be hard to show solid examples, so may not be necessary.